client/app/app.vue

```diff
@@ -18,14 +24,100 @@
                         svg: use(:xlink:href="require('./assets/sprites/icon-connected.svg')")
                         | {{ 'my_accounts connected title' | t }}
 
-        router-view
+        router-view(v-on:open-dialog='onOpenDialog')
+
 </template>
 
 
+<script>
+    const dialogs = [{
+        id: 'dialog-1',
+        hidden: false,
+        headerImage: 'test0.png',
+        success: {
+            route: { name: 'create-account-success' }
+        }
+    }]
+
+
+    export default {
+      data() {
+          return {
+              'dialogs': []
+          }
+      },
+
+      created () {
+          const query = this.$router.currentRoute.query
+
+          // Show Dialogs
+          if (query.dialog) {
+              const values = query.dialog.split(',')
+              values.forEach((value) => {
+                  this.onOpenDialog(value)
+              })
+          }
+      },
+
+      methods: {
+          onError (err) {
+
```

Comment:
My thoughs about the use of `Vue.config.errorHandler`: it's a smart and clever way to handle and log errors in the stack v2 logging system (poke @nono) when something goes wrong in the app. But to catch some feedback errors (like a misfilled field in a form) I think it's not the best way to do this. Probably having an `$emit` from the component, catched by `app.vue` that pop a notification is a better idea (which can be cumulated with the logging system)